src/core/loader/phaserDefaults.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * @copyright BBC 2020
+ * @author BBC Children's D+E
+ * @license Apache-2.0
+ */
+import { CANVAS_HEIGHT, CANVAS_WIDTH } from "../layout/metrics.js";
+import FontLoaderPlugin from "./font-loader/font-plugin.js";
+import { JSON5Plugin } from "./json5-loader/json5-plugin.js";
+import { ParticlesPlugin } from "./particles-loader/particles-plugin.js";
+import { getLauncherScreen } from "../debug/debug-screens.js";
+import * as debugMode from "../debug/debug-mode.js";
+import { Loader } from "./loader.js";
+import { Boot } from "./boot.js";
+import { getBrowser } from "../browser.js";
+import { getContainerDiv } from "./container.js";
+
+const getScenes = conf => Object.keys(conf).map(key => new conf[key].scene({ key, ...conf[key].settings }));
+
+export const getPhaserDefaults = config => {
+    const browser = getBrowser();
+    const scene = getScenes(Object.assign(config.screens, getLauncherScreen(debugMode.isDebug())));
+    scene.unshift(new Loader());
+    scene.unshift(new Boot(config.screens));
+
+    return {
+        ...{
```

Comment:
does something different happen if you do `{ ...{ props }, ...otherprops }` or is this the same as`{ props, ...otherprops }`?

Reply:
I've tended to do it by default because it's copying rather than mutating but in this case there's no point (since the object was just created anyway.

Reply:
Actually no what am I talking about. This is correct

{ props, ...otherprops }

will create a property called props with the object hanging off it.

Reply:
think my example was a bit confusing 😂